app/frontend/src/cljs/teet/meeting/meeting_view.cljs

```diff
@@ -157,53 +166,55 @@
 (defn meeting-view-container
   ([param]
    [meeting-view-container param theme-colors/gray-light])
-  ([{:keys [text-color content open? heading heading-button children after-children-component]
+  ([{:keys [text-color content open? heading heading-button children after-children-component on-open]
      :or {text-color :inherit
           open? false}}
     bg-color]
    (r/with-let [open? (r/atom open?)
                 toggle-open! #(do
+                                (when on-open (on-open))
```

Comment:
(on-open) is fired even when closing the view? Is this wanted or should it be just when it's opened?

Reply:
good question, I think it is and I should change the name to reflect that.